projects/wc/src/app/components/organization-management/organization-management.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, Input, ViewEncapsulation, inject } from '@angular/core';
+import { LuigiClient } from '@luigi-project/client/luigi-element';
+import { I18nService } from '@openmfp/portal-ui-lib';
+import {
+  ButtonComponent,
+  InputComponent,
+  LabelComponent,
+  OptionComponent,
+  SelectComponent,
+} from '@ui5/webcomponents-ngx';
+
+@Component({
+  selector: 'organization-management',
+  standalone: true,
+  imports: [
+    LabelComponent,
+    InputComponent,
+    ButtonComponent,
+    OptionComponent,
+    SelectComponent,
+  ],
+  templateUrl: './organization-management.component.html',
+  styleUrl: './organization-management.component.scss',
+  encapsulation: ViewEncapsulation.ShadowDom,
+})
+export class OrganizationManagementComponent {
+  private i18nService = inject(I18nService);
+
+  texts: any = {};
+  organizations: string[] = ['org1', 'org2'];
```

Comment:
I think at the moment its meant to have hard coded organizations here, I would just (for the users eye) write it out.
```suggestion
  organizations: string[] = ['Organization 1', 'Organization 2'];
```

Reply:
true & done